x/incentives/keeper/genesis.go

```diff
@@ -27,5 +43,6 @@ func (k Keeper) ExportGenesis(ctx sdk.Context) *types.GenesisState {
 		LockableDurations: k.GetLockableDurations(ctx),
 		Gauges:            k.GetNotFinishedGauges(ctx),
 		LastGaugeId:       k.GetLastGaugeID(ctx),
+		Groups:            k.GetAllGroups(ctx),
```

Comment:
We should create a separate entry for group gauges and put them in there instead of packaging them together with regular gauges.

We can also create a wrapper for groups and group gauges